presto-spi/src/main/java/io/prestosql/spi/StandardErrorCode.java

```diff
@@ -64,6 +64,47 @@
     INVALID_SPATIAL_PARTITIONING(0x0000_0029, USER_ERROR),
     INVALID_ANALYZE_PROPERTY(0x0000_002A, USER_ERROR),
     TYPE_NOT_FOUND(0x0000_002B, USER_ERROR),
+    CATALOG_NOT_FOUND(0x0000_002C, USER_ERROR),
+    SCHEMA_NOT_FOUND(0x0000_002D, USER_ERROR),
+    TABLE_NOT_FOUND(0x0000_002E, USER_ERROR),
+    COLUMN_NOT_FOUND(0x0000_002F, USER_ERROR),
+    ROLE_NOT_FOUND(0x0000_0030, USER_ERROR),
+    SCHEMA_ALREADY_EXISTS(0x0000_0031, USER_ERROR),
+    TABLE_ALREADY_EXISTS(0x0000_0032, USER_ERROR),
+    COLUMN_ALREADY_EXISTS(0x0000_0033, USER_ERROR),
+    ROLE_ALREADY_EXISTS(0x0000_0034, USER_ERROR),
+    NAMED_QUERY_ALREADY_EXISTS(0x0000_0035, USER_ERROR),
+    MISSING_COLUMN_NAME(0x0000_0036, USER_ERROR),
+    MISSING_CATALOG_NAME(0x0000_0037, USER_ERROR),
+    MISSING_SCHEMA_NAME(0x0000_0038, USER_ERROR),
+    INVALID_SCHEMA_NAME(0x0000_0039, USER_ERROR),
+    TYPE_MISMATCH(0x0000_003A, USER_ERROR),
+    INVALID_LITERAL(0x0000_003B, USER_ERROR),
+    COLUMN_TYPE_UNKNOWN(0x0000_003C, USER_ERROR),
+    MISMATCHED_COLUMN_ALIASES(0x0000_003D, USER_ERROR),
+    AMBIGUOUS_NAME(0x0000_003E, USER_ERROR),
+    INVALID_COLUMN_REFERENCE(0x0000_003F, USER_ERROR),
+    MISSING_GROUP_BY(0x0000_0040, USER_ERROR),
+    MISSING_ORDER_BY(0x0000_0041, USER_ERROR),
+    MISSING_OVER(0x0000_0042, USER_ERROR),
+    NESTED_AGGREGATION(0x0000_0043, USER_ERROR),
+    NESTED_WINDOW(0x0000_0044, USER_ERROR),
+    EXPRESSION_NOT_IN_DISTINCT(0x0000_0045, USER_ERROR),
+    TOO_MANY_GROUPING_SETS(0x0000_0046, USER_ERROR),
+    FUNCTION_NOT_WINDOW(0x0000_0047, USER_ERROR),
+    FUNCTION_NOT_AGGREGATE(0x0000_0048, USER_ERROR),
+    EXPRESSION_NOT_AGGREGATE(0x0000_0049, USER_ERROR),
+    EXPRESSION_NOT_SCALAR(0x0000_004A, USER_ERROR),
+    EXPRESSION_NOT_CONSTANT(0x0000_004B, USER_ERROR),
+    INVALID_ARGUMENTS(0x0000_004C, USER_ERROR),
+    TOO_MANY_ARGUMENTS(0x0000_004D, USER_ERROR),
+    INVALID_TRANSACTION_MODE(0x0000_004E, USER_ERROR),
```

Comment:
This can probably be `SYNTAX_ERROR`. It's only used when multiple isolation levels or read modes are specified, which is effectively a syntax error.